### PR TITLE
Draw trash icon

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: image-generator/yml/level-13/trash-finesse-two-cards.yml [hunk 1]
```diff
@@ -36,7 +36,9 @@ players:
     cards:
       - type: 1
         below: trash
```
Comment (Zamiell) [inline image-generator/yml/level-13/trash-finesse-two-cards.yml:38]: remove this

File: image-generator/yml/level-13/trash-finesse-two-cards.yml [hunk 2]
```diff
@@ -36,7 +36,9 @@ players:
...
+        trash: true
       - type: 1
         below: trash
```
Comment (Zamiell) [inline image-generator/yml/level-13/trash-finesse-two-cards.yml:41]: and this

File: image-generator/yml/level-13/trash-finesse.yml [hunk 3]
```diff
@@ -36,5 +36,6 @@ players:
       - type: 1
         below:
           text: trash
```
Comment (Zamiell) [inline image-generator/yml/level-13/trash-finesse.yml:38]: and this

File: image-generator/yml/level-15/unknown-trash-discharge.yml [hunk 4]
```diff
@@ -33,6 +33,7 @@ players:
       - type: x
       - type: r
         below: Trash
```
Comment (Zamiell) [inline image-generator/yml/level-15/unknown-trash-discharge.yml:35]: and this

File: image-generator/yml/level-4/chop-move-new.yml [hunk 5]
```diff
@@ -19,6 +19,7 @@ players:
         above: Chop
       - type: 1
         clue: 1
+        trash: true
         below:
```
Comment (Zamiell) [inline image-generator/yml/level-4/chop-move-new.yml:23]: and this

File: image-generator/yml/level-4/trash-chop-move-1.yml [hunk 6]
```diff
@@ -21,6 +21,7 @@ players:
           color: darkred
       - type: 1
         clue: 1
+        trash: true
         below:
```
Comment (Zamiell) [inline image-generator/yml/level-4/trash-chop-move-1.yml:25]: and this

File: image-generator/yml/level-4/trash-chop-move-2.yml [hunk 7]
```diff
@@ -15,6 +15,7 @@ players:
   - cards:
       - type: 1
         clue: 1
+        trash: true
         below:
```
Comment (Zamiell) [inline image-generator/yml/level-4/trash-chop-move-2.yml:19]: and this

File: image-generator/yml/level-4/trash-chop-move-3.yml [hunk 8]
```diff
@@ -16,13 +16,15 @@ players:
...
         below:
           text: Trash
           color: peru
       - type: x
         above: Chop
       - type: 1
         clue: 1
+        trash: true
         below:
           text: Trash
```
Comment (Zamiell) [inline image-generator/yml/level-4/trash-chop-move-3.yml:29]: and this

File: image-generator/yml/level-4/trash-chop-move-3.yml [hunk 9]
```diff
@@ -16,13 +16,15 @@ players:
       - type: x
       - type: 1
         clue: 1
+        trash: true
         below:
```
Comment (Zamiell) [inline image-generator/yml/level-4/trash-chop-move-3.yml:20]: and this

**Comment by DarthGandalf** [comment]
done

**Comment by Zamiell** [comment]
nice work

**Pull request merged**